doc/dev/conda_forge.rst

```diff
@@ -0,0 +1,28 @@
+.. include:: ../include/links.rst
+
+.. _conda_forge:
+
+How to manually update the PypeIt conda-forge feedstock (what to do if the bots are broken)
+===========================================================================================
+
+1. Fork `conda-forge/pypeit-feedstock <https://github.com/conda-forge/pypeit-feedstock>`_.
+2. Install required tools
+
+.. code-block bash::
+
+    conda install -c conda-forge conda-smithy
+
+3. Now manually update the version field in recipe/meta.yaml
+4. Update the source > sha256 hash in recipe/meta.yaml with the sha256 hash available on
```

Comment:
Not sure how `source > sha256` is intended to render.  Or if it all.
Please clarify

Reply:
I didn't intend `source > sha256` wasn't intended to render in any special way but I think `source.sha256` might be clearer as it is intended to represent

```yaml
source:
  sha256: hash_goes_here
```
in the `recipe/meta.yaml` file

Reply:
Got it!

In that case made do as you have done here but in .rst

You can use `::` to end a line and then indent the stuff below
to have it render sort of like what you have.  Albeit without color.

Like what we have in a doc like this:

https://github.com/pypeit/PypeIt/blob/master/doc/bias_dark.rst

Reply:
okay, I tested building the docs locally and the code blocks showed up right